### PR TITLE
Correcting: 'needs' instead of 'need'

### PR DESCRIPTION
**Pull request by jerzybrzoska**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ If the two blockheights drift apart it might be necessary to intervene.
...
 
-If you encrypt your `hsm_secret`, you will have to pass the `--encrypted-hsm` startup option to `lightningd`. Once your `hsm_secret` is encrypted, you __will not__ be able to access your funds without your password, so please beware with your password management. Also beware of not feeling too safe with an encrypted `hsm_secret`: unlike for `bitcoind` where the wallet encryption can restrict the usage of some RPC command, `lightningd` always need to access keys from the wallet which is thus __not locked__ (yet), even with an encrypted BIP32 master seed.
+If you encrypt your `hsm_secret`, you will have to pass the `--encrypted-hsm` startup option to `lightningd`. Once your `hsm_secret` is encrypted, you __will not__ be able to access your funds without your password, so please beware with your password management. Also beware of not feeling too safe with an encrypted `hsm_secret`: unlike for `bitcoind` where the wallet encryption can restrict the usage of some RPC command, `lightningd` always needs to access keys from the wallet which is thus __not locked__ (yet), even with an encrypted BIP32 master seed.
```
Comment (vincenzopalazzo) [inline README.md:220]:
```suggestion
If you encrypt your `hsm_secret`, you will have to pass the `--encrypted-hsm` startup option to `lightningd`. Once your `hsm_secret` is encrypted, you __will not__ be able to access your funds without your password, so please beware with your password management. Also, beware of not feeling too safe with an encrypted `hsm_secret`: unlike for `bitcoind` where the wallet encryption can restrict the usage of some RPC command, `lightningd` always needs to access keys from the wallet which is thus __not locked__ (yet), even with an encrypted BIP32 master seed.
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM.

Only a small change request

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/4835/commits/c7473a128c7d84652dae05ffe2fcb3587f4e5b90

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4835/commits/c7473a128c7d84652dae05ffe2fcb3587f4e5b90

**Comment by jerzybrzoska** [comment]
> LGTM.
> 
> Only a small change request

`lightningd` is singular third person, therefore the use of **needs** is warranted. I may be wrong, am I? [Reference](https://ell.stackexchange.com/questions/83380/can-i-use-needs-in-this-sentence).

**Comment by michaelfolkson** [comment]
I think the second commit adding a comma went wrong as it has co-authors. Regardless, I think there should only be one commit for a change like this and so squashing commits may be required.

ACK the change once it is a single commit.

**Comment by cdecker** [comment]
Squashed the commits into a single one

ACK https://github.com/ElementsProject/lightning/pull/4835/commits/8e4ef58e1ad775ec31e90982752bffa2fa4a7dfd

**Pull request merged**